gui/src/main/java/io/bisq/gui/main/market/offerbook/OfferBookChartView.java

```diff
@@ -88,7 +89,13 @@
     private HBox bottomHBox;
     private ListChangeListener<OfferBookListItem> changeListener;
     private ListChangeListener<CurrencyListItem> currencyListItemsListener;
-
+    private ChangeListener<Number> bisqWindowVerticalSizeListener;
+    private final double pixelsPerOfferTableRow = (109.0 / 4.0) + 10.0;
+    private final Function<Double, Double> offerTableViewHeight = (screenSize) -> {
+        // startup defaults: scene height = 710   tbl view height = 109   visible row count = 4
+        int extraRows = screenSize <= 710 ? 0 : (int) ((screenSize - 710) / pixelsPerOfferTableRow);
```

Comment:
Maybe you could move the initial layout height (710) as a field to the [Layout class](https://github.com/bisq-network/exchange/blob/master/gui/src/main/java/io/bisq/gui/util/Layout.java) and also replace it in the [BisqApp class](https://github.com/bisq-network/exchange/blob/1d0591084d6d2cc8204357204ca0f54b887ab765/gui/src/main/java/io/bisq/gui/app/BisqApp.java#L235) so the code will work as expected if we decide to change the default height at some point. Also please move the table view height (109) in a field and re-use it in the table view initialization https://github.com/bisq-network/exchange/blob/1d0591084d6d2cc8204357204ca0f54b887ab765/gui/src/main/java/io/bisq/gui/main/market/offerbook/OfferBookChartView.java#L302

Reply:
I pushed the changes you asked for in two commits that should have been one.